internal/codespaces/api/api.go

```diff
@@ -42,13 +43,13 @@ import (
 
 	"github.com/cenkalti/backoff/v4"
 	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghinstance"
+	"github.com/cli/cli/v2/pkg/cmdutil"
 	"github.com/opentracing/opentracing-go"
 )
 
 const (
-	githubServer = "https://github.com"
```

Comment:
`githubServer` and `vscsAPI` values were not in use anymore so I cleaned those up. I believe they were dead code from earlier versions of the Codespaces API.

Reply:
I ended up needing to add the server url back in to support the `--web` flows of list/create, but the `vscsAPI` is no longer needed.